pkg/azurefile/nodeserver.go

```diff
@@ -59,6 +59,19 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 	mountPermissions := d.mountPermissions
 	context := req.GetVolumeContext()
 	if context != nil {
+		// token request
+		if context[serviceAccountTokenField] != "" && context[clientIDField] != "" {
+			klog.V(2).Infof("NodePublishVolume: volume(%s) mount on %s with service account token, VolumeContext: %v", volumeID, target, context)
+			_, err := d.NodeStageVolume(ctx, &csi.NodeStageVolumeRequest{
```

Comment:
seems it could be combined with L83

Reply:
L83 is for ephemeral volume, I think L63 is also for ephemeral volume, right?

Reply:
token request can support both static pv and ephemeral volume.

Reply:
I think handle these two scenerios seperately here would be more clear to read the code. If we combine them it may increase the complexities. WDYT?

Reply:
ephemeral volume mount would only call `NodePublushVolume` func, so that's the reason why it has to call NodeStageVolume inside `NodePublushVolume` func;
if it's not for ephemeral volume, the `serviceAccountToken` should be passed into NodeStageVolume, and handle the static mount in `NodeStageVolume` only, `NodePublushVolume` should only do bind mount

Reply:
> token request can support both static pv and ephemeral volume.

so if you only add feature code support in `NodeStageVolume`, it should work for both static pv and ephemeral volume mounts

Reply:
> if it's not for ephemeral volume, the serviceAccountToken should be passed into NodeStageVolume, and handle the static mount in NodeStageVolume only, NodePublushVolume should only do bind mount

serviceAccountToken is also only sent in NodePublishVolume. So, I handled similar to ephemeral volume.